src/wp-includes/class-wp-network-query.php

```diff
@@ -111,27 +111,30 @@ class WP_Network_Query {
 	 *     @type string[]     $path__not_in         Array of paths to exclude affiliated networks for. Default empty.
 	 *     @type string       $search               Search term(s) to retrieve matching networks for. Default empty.
 	 *     @type bool         $update_network_cache Whether to prime the cache for found networks. Default true.
```

Comment:
These descriptions need to be realigned.

Reply:
Fixed.